api/api/src/main/scala/uk/ac/wellcome/platform/api/services/ImagesService.scala

```diff
@@ -18,21 +18,21 @@ class ImagesService(searchService: ElasticsearchService,
   private val nVisuallySimilarImages = 5
   private val imagesRequestBuilder = new ImagesRequestBuilder(queryConfig)
 
-  def findImageById(id: String)(
-    index: Index): Future[Either[ElasticError, Option[AugmentedImage]]] =
+  def findImageById(id: String)(index: Index)
+    : Future[Either[ElasticError, Option[Image[ImageState.Indexed]]]] =
```

Comment:
In the `WorkState` stuff we tend to import the ADT types so we can write `Work[Denormalised]` etc, which I find reads a bit nicer and is easier to parse in code with lots of type parameters around. Do you think that would get confusing doing that for `ImageState` when possible (where there are no name clashes), or do you think it would get confusing due to having some of the same names?

Reply:
I actually started off importing the types and then changed to this, because as you say it got confusing due to name clashes.

Reply:
:+1: that's annoying, ah well can't have it all

Reply:
Is this true? [Searching through the code](https://github.com/wellcomecollection/catalogue/search?p=2&q=WorkState.) we seem to use a mixture of both?